src/C1T2X_OBU.py

```diff
@@ -169,6 +186,20 @@ def LAN_listening_thread():
 			if pkt:
 				if not parseLANPacket:
 					sendVANET(pkt[0])
+					# Wait for ack
+					waiting_for_ack = True
+					c1t2x_logger.info("Message sent, waiting for ack")
+					time.sleep(1.0)
+					for i in range(120):  # Attempt to rebroadcast for 2 minutes before giving up
+						with mutex:
+							if waiting_for_ack:
+								sendVANET(pkt[0])
+								c1t2x_logger.info("Still waiting for ack")
+							else:
+								break
+						time.sleep(1.0)
+                    if waiting_for_ack:
+                        raise Exception("Ack was never received")
 				else:
```

Comment:
This if condition checks if we exiting the rebroadcasting loop because we received an ack or the timeout instance was reached. So, if we exited the above loop and `waiting_for_ack` is still true, then we gave up because the other Pi never sent back an ack.

Reply:
Got it, it was just another part of the indentation issue